tasks/eva_3156/eva_schema.json

```diff
@@ -0,0 +1,321 @@
+{
+  "version": "1.0.0",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "id": "EVA_metadata_schema",
+  "author": "EVA",
+  "type": "object",
+  "title": "EVA metadata schema",
+  "definitions": {
+    "EVABioSample": {
+
+    }
+  },
+  "description": "",
+  "required": [
+    "submitterDetails",
+    "project",
+    "sample",
+    "analysis",
+    "files"
+  ],
+  "properties": {
+    "submitterDetails": {
+      "type": "array",
+      "description": "Credentials of the submitter",
```

Comment:
Since "credentials" is an overloaded term in tech, may be we can say something like "Contact details" instead?